javascript/net/grpc/web/generator/BUILD.bazel

```diff
@@ -3,6 +3,7 @@ load("@rules_cc//cc:defs.bzl", "cc_binary")
 cc_binary(
     name = "protoc-gen-grpc-web",
     srcs = [
+        "version.h",
```

Comment:
this probably needs to be in the `hdrs` section:
https://docs.bazel.build/versions/main/be/c-cpp.html#cc_library

Reply:
If it's too much trouble, i don't mind if you inline this into the cc file too :)

Reply:
Seems like this does not apply to `cc_binary` (see third paragraph):
cc_binary and cc_test rules do not have an exported interface, so they also do not have a hdrs attribute. All headers that belong to the binary or test directly should be listed in the srcs.